tasks/skylab/TagSortBam.wdl

```diff
@@ -3,13 +3,14 @@ version 1.0
 task CellSortBam {
   input {
     File bam_input
+    File original_gtf
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
-    Int machine_mem_mb = 100000
-    Int cpu = 2
-    Int disk = ceil(size(bam_input, "Gi") * 8)
-    Int preemptible = 3
+    String docker = "quay.io/kishorikonwar/secondary-analysis-python3-scientific:sctools-optimized8"
+    Int machine_mem_mb = 8000
+    Int cpu = 8
+    Int disk = ceil(size(bam_input, "Gi") * 4)
+    Int preemptible = 0
```

Comment:
how long does this take? definitely set this to 0 if it consistently takes more than 24 hrs to run , otherwise consider giving at least 1 preemptible

Reply:
depends on the data size which varies widely